docs/troubleshoot/minimal.md

```diff
@@ -76,15 +76,39 @@ Migration progress can be tracked by monitoring the Dataflow job and following c
 
 It can happen that in retryDLQ mode, there are still permanent errors. To identify that all the retryable errors have been processed and only permanent errors remain for reprocessing - one can look at the ‘Successful events' count - it would remain constant after every retry iteration. Each retry iteration, the ‘elementsReconsumedFromDeadLetterQueue' would increment.
 
-### Retry command
+### Retry commands
+
+#### To rerun regular flow
```

Comment:
Use re-run mode to keep it consistent everywhere

Reply:
done